actor/builtin/miner/miner_test.go

```diff
@@ -1178,107 +1187,145 @@ func assertSlashStatus(t *testing.T, st state.Tree, vms vm.StorageMap, minerAddr
 func TestVerifyPIP(t *testing.T) {
 	tf.UnitTest(t)
 
-	ctx := context.Background()
-	st, vms := th.RequireCreateStorages(ctx, t)
-
-	ancestors := th.RequireTipSetChain(t, 10)
-
-	origPid := th.RequireRandomPeerID(t)
-	minerAddr := th.CreateTestMiner(t, st, vms, address.TestAddress, origPid)
-
-	sectorId := uint64(1)
-	commD := th.MakeCommitment()
+	sectorSetWithOneCommitment, commDForSectorIdOne := func() (SectorSet, types.CommD) {
+		set := NewSectorSet()
+		comms := th.MakeCommitments()
+		set.Add(1, comms)
```

Comment:
nit: to make this a bit clearer I would give `1` a variable name so you can pass that into `VerifyPieceInclusion` and lead readers back to the setup.

Reply:
Done! Your wish is my command (to Sidney).